sources/packages/api/src/route-controllers/cra-integration/models/process-response.res.dto.ts

```diff
@@ -1,4 +1,4 @@
-export interface ProcessResponseResDto {
+export class ProcessResponseResAPIOutDTO {
```

Comment:
Can we rename this file to remove the `.res`?
I believe that it was an old pattern that we did not proceed using it.

Reply:
Can we remove the "Res" from the name also? "Response+Res" seems too much.